openshift_cli_installer/libs/managed_clusters/helpers.py

```diff
@@ -41,6 +40,17 @@ def prepare_managed_clusters_data(
                 _cluster["machine-cidr"] = _cluster.get("cidr", "10.0.0.0/16")
 
         elif cluster_platform == GCP_OSD_STR:
+            if gcp_service_account_file:
```

Comment:
please remove, this is checked in https://github.com/RedHatQE/openshift-cli-installer/blob/main/openshift_cli_installer/utils/cli_utils.py#L472